src/toil/cwl/cwltoil.py

```diff
@@ -1815,7 +1817,10 @@ def determine_load_listing(tool: ToilCommandLineTool):
                            --logInfo \\
                            example.cwl \\
                            example-job.yaml
-            """[1:])
+            """[
+        1:
+    ]
```

Comment:
Is this really the best indentation here?

Reply:
I don't like it either, but this is the result of `Black` and running `make format`.